library/Vanilla/Models/ThemeModel.php

```diff
@@ -371,9 +372,14 @@ public function verifyThemeIdentifierIsValid($themeKey) {
     /**
      * Get current theme.
      *
+     * @param bool $prioritizeAddon We have to decide if we want an accurate addon or not.
+     *
+     * If false (default) - The Garden.CurrentTheme will take priority on mobile.
+     * If true - The Garden.MobileTheme will take priority on mobile.
+     *
      * @return array The current theme or the fallback if it fails to load.
      */
-    public function getCurrentTheme(): array {
+    public function getCurrentTheme(bool $prioritizeAddon = false): array {
```

Comment:
You may want to consider a string enum with a constant on this class rather than a bool. When the bool isn't clear to the reader, I prefer an enum.

Reply:
Done. I've made 2 constants.

Definitely cleaner, and gives me a better spot to put longer descriptions.